variables.tf

```diff
@@ -111,14 +111,14 @@ variable "ami_type" {
   type        = string
   description = <<-EOT
     Type of Amazon Machine Image (AMI) associated with the EKS Node Group.
```

Comment:
Have you tested this change?

I think you may need more changes to this file

https://github.com/cloudposse/terraform-aws-eks-node-group/blob/ea8401ef0db1cce91fd96ce1776407f552dbc090/ami.tf#L4

Reply:
Indeed I missed the file.
I not tested properly, I was in rush working locally with quick tricks to make my need work.

I will take a look better, thanks for catching that!